src/Plutarch/Orphans.hs

```diff
@@ -172,6 +174,12 @@ deriving anyclass instance Aeson.ToJSON TxOutRef
 -- @ since 3.6.1
 deriving anyclass instance Aeson.FromJSON TxOutRef
 
+-- @ since 3.15.4
+deriving anyclass instance Aeson.ToJSON Credential
+
+-- @ since 3.15.4
+deriving anyclass instance Aeson.FromJSON Credential
```

Comment:
`Generic` derivations for Aeson are horribly slow, horribly opaque, and horribly brittle. I would recommend against them.